components/03-modules/mega-menu/mega-menu.config.js

```diff
@@ -0,0 +1,177 @@
+            link: '/example',
+            elements: [
+              {
+                class: 'mega-menu__inner-item mega-menu__inner-item--level2',
+                text: 'Lorem ipsum',
+                link: '/lorem-ipsum',
+                elementLink: '#',
+                linkClass: 'mega-menu__link mega-menu__link--inner'
+              },
+              {
+                class: 'mega-menu__inner-item mega-menu__inner-item--level2',
+                text: 'Lorem ipsum',
+                link: '/lorem-ipsum',
+                elementLink: '#',
+                linkClass: 'mega-menu__link mega-menu__link--inner'
+              },
+              {
+                class: 'mega-menu__inner-item mega-menu__inner-item--level2',
+                text: 'Lorem ipsum',
+                link: '/lorem-ipsum',
+                elementLink: '#',
+                linkClass: 'mega-menu__link mega-menu__link--inner'
+              },
+              {
+                class: 'mega-menu__inner-item mega-menu__inner-item--level2',
+                text: 'Lorem ipsum',
+                link: '/lorem-ipsum',
+                elementLink: '#',
+                linkClass: 'mega-menu__link mega-menu__link--inner'
+              }
+            ]
+          }
+        ],
+        imageTitle: 'Lorem Ipsum',
+        secondImageTitle: 'lorem ipsum'
+      },
+      {
+        text: 'Accessories',
+        link: '/accessories'
+      },
+      {
+        class: 'mega-menu__item--parent',
+        text: 'TBD',
+        link: '/tbd',
+        nestedList: [
+          {
+            class: '',
+            heading: 'Example',
+            link: '/example',
+            elements: [
+              {
+                class: '',
+                text: 'Lorem ipsum',
+                link: '/lorem-ipsum',
+                elementLink: '#',
+                linkClass: 'mega-menu__link mega-menu__link--inner'
+              },
+              {
+                class: '',
+                text: 'Lorem ipsum',
+                link: '/lorem-ipsum',
+                elementLink: '#',
+                linkClass: 'mega-menu__link mega-menu__link--inner'
+              },
+              {
+                class: '',
+                text: 'Lorem ipsum',
+                link: '/lorem-ipsum',
+                elementLink: '#',
+                linkClass: 'mega-menu__link mega-menu__link--inner'
+              },
+              {
+                class: '',
+                text: 'Lorem ipsum',
+                link: '/lorem-ipsum',
+                elementLink: '#',
+                linkClass: 'mega-menu__link mega-menu__link--inner'
+              }
+            ]
+          },
+          {
+            class: '',
+            heading: 'Example1',
+            link: '/example1',
+            elements: [
+              {
+                class: '',
+                text: 'Lorem ipsum',
+                link: '/lorem-ipsum',
+                elementLink: '#',
```

Comment:
we have `link` and `elementLink` I cannot find usage of `elementLink` in while project. could you please check it and remove if not used?

Reply:
Yes, you're right. I've removed it.